lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java

```diff
@@ -331,6 +331,60 @@ void testUpdateCallbackInfoOfSimpleTask()
 
     Task retrievedUpdatedTask = taskService.getTask(createdTask.getId());
 
-    assertEquals(callbackInfo, retrievedUpdatedTask.getCallbackInfo());
+    assertThat(retrievedUpdatedTask.getCallbackInfo()).isEqualTo(callbackInfo);
+  }
+
+  @WithAccessId(
+      userName = "user_1_2",
+      groupNames = {"group_1"})
+  @Test
+  void testSetOwnerAndSubsequentClaimSucceeds()
+      throws TaskNotFoundException, NotAuthorizedException, InvalidStateException,
+          InvalidOwnerException {
+
+    TaskService taskService = taskanaEngine.getTaskService();
+    String taskReadyId = "TKI:000000000000000000000000000000000025";
+    Task taskReady = taskService.getTask(taskReadyId);
+    assertThat(taskReady.getState()).isEqualTo(TaskState.READY);
+    assertThat(taskReady.getOwner()).isNull();
+    Task modifiedTaskReady = taskService.setOwner(taskReadyId, "Holger");
```

Comment:
why Holger? 
Has Holger any meaning like an existing account with access rights? Or is this just any name? Is Holger part of the dataset?
Can we clarify this and extract this to a variable? Like String anyUserName = "Holger"?